views/peeps/peeps.erb

```diff
@@ -0,0 +1,5 @@
+<ul>
+  <% @peeps.reverse.each do |peep| %>
```

Comment:
good choice to use reverse, however it might be good to check that @peeps is definitely keeping the peeps in chronological order to begin with